docs/README.md

```diff
@@ -0,0 +1,20 @@
+Welcome to the libsass wiki!
+
+## First Off
+LibSass is just a library. To run the code locally (i.e. to compile your stylesheets), you need an implementer. SassC (get it?) is an implementer written in C. There are a number of other implementations of LibSass - for example Node. We encourage you to write your own port - the whole point of Libsass is that we want to bring Sass to many other languages, not just Ruby!
+
+We're working hard on moving to full parity with Ruby Sass... learn more at the [The-LibSass-Compatibility-Plan](compatibility-plan.md)!
+
+### Implementing libsass
+
+If you're interested in implementing libsass in your own project see the [API Documentation](api-doc.md) which now includes implementing 
+your own [Sass functions](api-function.md).  You may wish to [look at other implementations](implementations.md) for your language of choice. 
+Or make your own!
+
+### Contributing to libsass
+
+|   Issue Tracker   |            Issue Triage          |     Community Guidelines    |
+|-------------------|----------------------------------|-----------------------------|
+| We're always needing help, so checkout our issue tracker, help some people out, and read our article on [Contributing](contributing.md)! It's got all the details on what to do! | To help understand the process of triaging bugs, have a look at our [Issue-Triage](triage.md) document. | Oh, and don't forget we always follow [[Sass Community Guidelines|http://sass-lang.com/community-guidelines]]. Be nice and everyone else will be nice too! |
```

Comment:
I think you want "check out" here.
http://grammarist.com/spelling/checkout-check-out/